docs/source/governance.rst

```diff
@@ -1,37 +0,0 @@
-PyTorch-Ignite governance
```

Comment:
We do not have any equivalent document on the new site. We have to move it there before deleting here...